reference/orders.v2.oas2.yml

```diff
@@ -2580,7 +2580,7 @@ components:
                 type: number
               date_created:
                 type: string
-                description: 'The date this order was created. If not specified, will default to the current time. The date should be in RFC 2822 format, e.g.: Tue, 20 Nov 2012 00:00:00 +0000'
+                description: 'The date the order was created, formatted in the RFC-2822 standard. This attribute is set on Order creation (POST) (to support migration of historical orders). If no value is provided, then it will default to the current date/time. e.g. `Tue, 20 Nov 2012 00:00:00 +0000`'
```

Comment:
The date the order was created, formatted in the RFC-2822 standard. You set this attribute on Order creation (POST) to support the migration of historical orders. If you do not provide a value, then it will default to the current date/time. e.g., Tue, 20 Nov 2012 00:00:00 +0000'